### PR TITLE
Rename FileStreamSignature to StreamFileSignature

### LINKED ISSUE #4595: Rename FileStreamSignature to StreamFileSignature
### Problem

The class `FileStreamSignature` represents a signature file associated with a record file, balance, file or event file. Since we normally call those "stream files" and they implement an interface `StreamFile`, it makes sense the signature associated with them be named `StreamFileSignature`.

### Solution

* Rename `FileStreamSignature` to `StreamFileSignature`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:

Rename `FileStreamSignature` to `StreamFileSignature`

**Related issue(s)**:

Fixes #4595

**Notes for reviewer**:
No logic change, just a simple renaming done through the IDE

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/ConsensusValidatorImpl.java [hunk 1]
```diff
@@ -26,10 +26,12 @@
 import java.math.RoundingMode;
 import java.util.Collection;
 import javax.inject.Named;
+
+import com.hedera.mirror.importer.domain.StreamFileSignature;
+
```
Comment (edwin-greene) [inline hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/ConsensusValidatorImpl.java:31]: Nit: Looks like it needs an optimize imports.

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/NodeSignatureVerifier.java [hunk 2]
```diff
@@ -23,11 +23,13 @@
 import java.security.Signature;
 import java.util.Collection;
 import javax.inject.Named;
+
+import com.hedera.mirror.importer.domain.StreamFileSignature;
+
```
Comment (edwin-greene) [inline hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/NodeSignatureVerifier.java:28]: Nit: optimize.

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/CompositeSignatureFileReader.java [hunk 3]
```diff
@@ -23,11 +23,13 @@
 import java.io.DataInputStream;
 import java.io.IOException;
 import javax.inject.Named;
+
+import com.hedera.mirror.importer.domain.StreamFileSignature;
+
```
Comment (edwin-greene) [inline hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/CompositeSignatureFileReader.java:28]: Nit: optimize.

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/SignatureFileReaderV5.java [hunk 4]
```diff
@@ -24,11 +24,13 @@
 
 import java.io.IOException;
 import javax.inject.Named;
+
+import com.hedera.mirror.importer.domain.StreamFileSignature;
+
```
Comment (edwin-greene) [inline hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/SignatureFileReaderV5.java:29]: Nit: optimize.

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/ConsensusValidatorImpl.java (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/ConsensusValidatorImpl.java]: Thanks, optimized

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/NodeSignatureVerifier.java (hunk 2 above)
Comment (steven-sheehy) [reply hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/NodeSignatureVerifier.java]: optimized

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/CompositeSignatureFileReader.java (hunk 3 above)
Comment (steven-sheehy) [reply hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/CompositeSignatureFileReader.java]: optimized

File: hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/SignatureFileReaderV5.java (hunk 4 above)
Comment (steven-sheehy) [reply hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/reader/signature/SignatureFileReaderV5.java]: optimized

**Review by edwin-greene — APPROVED** [review]
Looks good.

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**